package.json

```diff
@@ -1,11 +1,27 @@
 {
-  "name": "Cuttle",
-  "private": true,
+  "name": "cuttle",
   "version": "3.0.2",
+  "private": true,
   "description": "The deepest card game under the sea",
-  "engines": {
-    "node": "16.13.2"
+  "scripts": {
+    "build": "vue-cli-service build && cp ./client/public/favicon.ico ./assets",
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint --no-fix",
+    "debug": "node debug app.js",
+    "e2e:client": "npx cypress run --config-file tests/cypress.config.js",
+    "e2e:gui": "ENABLE_VUE_DEVTOOLS=true npx cypress open --config-file tests/cypress.config.js",
+    "e2e:mockprod": "CUTTLE_ENV=production npm run e2e:server",
+    "e2e:server": "VUE_APP_API_URL=http://localhost:1337 npx cypress run --config-file tests/cypress.config.js",
+    "format:js": "./scripts/format-project-js",
+    "lint:fix": "vue-cli-service lint",
+    "start": "node app.js",
+    "start:client": "concurrently \"ENABLE_VUE_DEVTOOLS=true npm run start:client\" \"npm run start:devtools\"",
```

Comment:
Oops, this needs to be `"start:client": "concurrently \"ENABLE_VUE_DEVTOOLS=true vue-cli-service serve\" \"npm run start:devtools\"",` or it will recursively call itself infinitely